.github/workflows/deploy.yml

```diff
@@ -0,0 +1,50 @@
+name: deploy
+
+on: [pull_request]
+
+env:
+    DEPLOY_USER: omniledger
+    DEPLOY_HOST: srv1.c4dt.org
+
+jobs:
+    deploy-website:
+        name: deploy website
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+
+            - uses: kielabokkie/ssh-key-and-known-hosts-action@v1.2.0
+              with:
+                  ssh-host: ${{ env.DEPLOY_HOST }}
+                  ssh-private-key: ${{ secrets.DEPLOY_SSH_KEY }}
+
+            - name: update remotely
+              run: >
+                  ssh "${{ env.DEPLOY_USER }}@${{ env.DEPLOY_HOST }}"
+                  bin/update.sh
+
+    prerelease-dynacred:
+        name: prerelease dynacred
+        runs-on: ubuntu-latest
+        defaults:
+            run:
+                working-directory: dynacred
+        steps:
+            - uses: actions/checkout@v2
+
+            - uses: actions/setup-node@v1
+              with:
+                  node-version: 12
+                  cache: npm
+
+            - run: npm install
+
+            - run: npm test
```

Comment:
I'm confused - shouldn't the test be run _before_ all else?

Reply:
It's ran in the "lint and test" workflow